lib/install/elm.rb

```diff
@@ -19,4 +19,7 @@
 run "./bin/yarn add --dev elm-hot-loader elm-webpack-loader"
 run "yarn run elm package install -- --yes"
 
+puts "Updating elm source location"
+gsub_file "elm-package.json", /\"\.\"\n/, %("app/javascript/packs"\n)
```

Comment:
You will get relative paths like so - `"app/assets/javascript/entries"` or `"app/assets/packs"`